.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: ruby
+sudo: false
+cache: bundler
+script:
+  - "bundle exec rspec spec"
```

Comment:
`rspec spec` is not a PL thing and it just runs all the spec tests within the `./spec` directory.
There's a common pattern in projects here at PL that instead define a `:spec` task in the project's Rakefile, which usually just does some project-specific setup and then calls `rspec spec` basically.

If you look at the [Rakefile](https://github.com/puppetlabs/ruby-hocon/blob/master/Rakefile) in this repo, it's pretty minimal but it defines a `:spec` task and sets the default task to it (so `rake` => `rake spec`).

I would probably change this to `bundle exec rake spec` since that's more conventional here, plus it'll be easier to extend the spec testing behavior in the future by modifying the Rakefile without having to change this travis file.